docs/source.md

```diff
@@ -24,7 +24,7 @@
 3. Move the resulting `bin/gh` executable to somewhere in your PATH
 
    ```sh
-   $ sudo mv ./bin/gh /usr/local/bin/
+   $ sudo mv ./bin/gh /usr/bin/
```

Comment:
This was just an example and it was meant to suggest `/usr/local/bin` as a destination. Please revert this change 👍